ci/quarkus.Jenkinsfile

```
@@ -48,6 +48,9 @@ pipeline {
 					}
 					dir('quarkus') {
 						sh "git clone -b 3.15 --single-branch https://github.com/quarkusio/quarkus.git . || git reset --hard && git clean -fx && git pull"
+						// This is a temporary workaround because Quarkus reverted the Hibernate ORM and Reactive upgrade.
+						// Remove this once Quarkus upgrades the versions again
+						sh "git reset --hard f42166ee7041ed09b7183d5dbf3ece2439b16676"
```

Not sure what resetting has to do with this, I'd have gone with a switch?

```suggestion
						sh "git switch --detach f42166ee7041ed09b7183d5dbf3ece2439b16676"
```

---

Using reset was the first suggestion that showed up when I searched for this. I doubt that `switch` makes a difference.